.travis.yml

```diff
@@ -1,32 +1,80 @@
-os: linux
-dist: trusty
-sudo: required
-language: python
-python:
-  - "2.7"
-  - "3.6"
-
-addons:
-  apt:
-    packages:
-      - dos2unix
+language: generic
+
+matrix:
+  include:
```

Comment:
open question: how many of these configurations do we actually want to run? I've added a macos version of everything, which doubles the number of configurations from 8 to 16. Do we want to add just one or two instead?

Reply:
Basically we want to add as many as we can without causing too much of a Travis CI backlog. My guess is that more than 5 configurations will cause jams, but that's more a feeling than something experimentally verified ;)

Reply:
++